serverlessworkflow/modules/ROOT/pages/cloud/common/_proc_deploy_sw_kn_cli.adoc

```diff
@@ -0,0 +1,57 @@
+[[proc-deploy-sw-application-knative-cli]]
+=== Deploying your workflow application using Knative CLI
+
+Once you have pushed your workflow application into the {registry} registry, you can use the command-line tools, such
+as Knative CLI or `kubectl` to initiate the deployment process.
+
+.Prerequisites
+* Workflow application is installed.
```

Comment:
what does it mean ?

Reply:
IIRC it was update by @kaldesai as well in when it was merge in the past.
I guess it means that the workflow and the container image is built.